src/Twee/WorkInProgress/School Slut Good Ending Scene 1_1.twee

```diff
@@ -0,0 +1,22 @@
+:: School Slut Good Ending Scene 1_1
+<div style="float: right; margin: 10px;"><<RandomImageFromPack "spa_reception" 400 400>></div>\
+Having arrived at the designated address, you look at the plain business card, double checking you’re in the correct place. The appointment time printed on the back is fast approaching, and you nervously wonder what $teacher has planned for you inside...
+
+"The Pink Aster Spa" it says on the marquee, cheery flowers decorating the front of the shop combined with a UV tint to obscure the interior. You take a final glance around the strip mall - everything so far is fairly innocuous and you see no sign of $teacher. Remembering her warning about standing her up you take a deep breath and heading inside.
+
+A merry bell jingles as you open the door and you are greeted by the sweet smell of flowery perfume, the two women at the front desk looking up to greet you with friendly, dazzling smiles.
+
+@@.femvoice2;"Hello, welcome to The Pink Aster - your ultimate destination for relaxing luxury,"@@ the two women recite together. The practiced synchronicity is a little creepy but short lived as only one of the women continues on past the canned introduction. 
+@@.femvoice2;"Would you like to make an appointment? We are offering a new member introductory special this month for only $500."@@ 
+
+You can't help but blink at the price. This must be a fancy place indeed if that's the sale price... Realizing that both women are awaiting your response, you take a handful of steps to bring you to the counter, explaining that, actually, you have an appointment? Maybe under $teacher?
+
+Looking down one of the girls taps away at her computer for a few moments and then looks up with a wide smile. @@.femvoice2;"Ah, here we go! Miss $player.name, scheduled for our Deluxe "Pampered Princess" total treatment package?"@@
+
+<<if $player.femaleName>>You blush, even after all this time with your "new" name, hearing it so naturally fall from a stranger's lips makes you shiver,<<else>>You blink at the name of the package, <<endif>>wondering what would warrant $teacher to spring for something so fancy that it could be worth more than your debt payments combined. Feeling nervous and undeserving, you nevertheless nod.
```

Comment:
```suggestion
<<if $player.femaleName>>You blush, even after all this time with your "new" name, hearing it so naturally fall from a stranger's lips makes you shiver, <<else>>You blink at the name of the package, <<endif>>wondering what would warrant $teacher to spring for something so fancy that it could be worth more than your debt payments combined. Feeling nervous and undeserving, you nevertheless nod.
```